gluten-core/src/main/scala/org/apache/spark/sql/execution/datasources/GlutenWriterColumnarRules.scala

```diff
@@ -80,48 +74,53 @@ case class FakeRowAdaptor(child: SparkPlan) extends UnaryExecNode with GlutenPla
     copy(child = newChild)
 }
 
-object GlutenColumnarRules {
-
-  // TODO: support Insert clause
-
+object GlutenWriterColumnarRules {
   // TODO: support ctas in Spark3.4, see https://github.com/apache/spark/pull/39220
   // TODO: support dynamic partition and bucket write
   //  1. pull out `Empty2Null` and required ordering to `WriteFilesExec`, see Spark3.4 `V1Writes`
   //  2. support detect partition value, partition path, bucket value, bucket path at native side,
   //     see `BaseDynamicPartitionDataWriter`
-  def isGlutenInsertInto(cmd: DataWritingCommand): Boolean = {
+  def isNativeParquetAppliable(cmd: DataWritingCommand): Boolean = {
```

Comment:
isNativeParquetAvailable?

Reply:
native parquet is always available , but in some cases it's not applicable, that's why we need to check it.